ynr/apps/bulk_adding/helpers.py

```diff
@@ -4,21 +4,19 @@
                                raise_if_unsafe_to_delete)
 from candidates.models.auth import check_creation_allowed
 from candidates.views.version_data import get_change_metadata, get_client_ip
+from tasks.models import connect_task_signal, disconnect_task_signal
 
 
 def add_person(request, person_data):
+    disconnect_task_signal()
```

Comment:
Is there any risk here that we might disconnect at the top of this function, throw an exception somewhere in the body and then not re-connect at the end, leaving us in an inconsistent state? Would it be better to

```python
disconnect_task_signal()
try:
    # do stuff
finally:
    connect_task_signal()
```